### PR TITLE
Clusterbootstrap and AntreaConfig feature gate checks

### PR DESCRIPTION
**Pull request by knabben**
### What this PR does / why we need it
Changes in the clusterbootstrap and antreaconfig to allow adaption with user variables

### Which issue(s) this PR fixes

Fixes #

### Release note

```release-note

```

### CODE REVIEW

**Comment by alfredthenarwhal** [comment]
Tests failed! Build no: 3162

**Comment by alfredthenarwhal** [comment]
> @rajaskakodkar:  /test install-vc7
Commit: d551672

Tests failed! Build no: 3169

**Comment by alfredthenarwhal** [comment]
> @rajaskakodkar:  /test install-vc7
Commit: d551672

Tests canceled! Build no: 3174

**Comment by alfredthenarwhal** [comment]
> @rajaskakodkar:  /test install-vc7
Commit: d551672

Build failed! Build no: 3176

File: providers/yttcb/clusterbootstrap.yaml [hunk 1]
```diff
@@ -74,6 +79,31 @@ metadata:
...
+        idleFlowTimeout: #@ data.values.ANTREA_FLOWEXPORTER_IDLE_TIMEOUT
+      kubeAPIServerOverride: #@ data.values.ANTREA_KUBE_APISERVER_OVERRIDE
+      transportInterface: #@ data.values.ANTREA_TRANSPORT_INTERFACE
+      transportInterfaceCIDRs: #@ split_comma_values(data.values.ANTREA_TRANSPORT_INTERFACE_CIDRS)
+      multicastInterfaces: #@ split_comma_values(data.values.ANTREA_MULTICAST_INTERFACES)
+      tunnelType: #@ data.values.ANTREA_TUNNEL_TYPE
+      trafficEncryptionMode: #@ data.values.ANTREA_TRAFFIC_ENCRYPTION_MODE
+      enableUsageReporting: #@ data.values.ANTREA_ENABLE_USAGE_REPORTING
+      wireGuard:
+        port: #@ data.values.ANTREA_WIREGUARD_PORT
```
Comment (12345lcr) [inline providers/yttcb/clusterbootstrap.yaml:105]: Should these values above be the part of `antrea_configs_exist()`?

**Review by jeffwubj — APPROVED** [review]

File: providers/yttcb/clusterbootstrap.yaml (hunk 1 above)
Comment (knabben) [reply providers/yttcb/clusterbootstrap.yaml:105]: I do prefer to keep only the feature flags, other values does not mean much.

we should even consider data.values.cni instead, if the default exists and other CNI is picked the config will be created.

**Review by 12345lcr — APPROVED** [review]

**Comment by knabben** [comment]
This was rebased in the Antrea bump pr

**Pull request closed without merging**